File: linked.py
#!/usr/bin/env python3

class LinkedList:
    def _init_(self):
        self.nodes = []

    def append(self, node):
        self.nodes.append(node)

    def get(self, value):
        for node in self.nodes:
            if node.get() == value:
                return node

        return None

class Node:
    def _init_(self, value, next=None):
        self.value = value
        self.next = next

    def get(self):
        return self.value

    def next_node(self):
        return self.next

File: linked.js
class LinkedList {
  constructor() {
    this.nodes = []
  }

  getNode(value){
    return this.nodes.filter(node => node.value == value)
  }

  append(node){
    this.nodes.unshift(node)
  }

  as_json(){
    return JSON.stringify(this.nodes)
  }
}

class Node {
  constructor(value, next) {
    this.value = value
    this.next  = next
  }

   value(){
    return this.value
  }

   nextNode(){
    return this.next
  }
}
